app/services/proofing/aamva/request/templates/verify.xml.erb

```diff
@@ -1,4 +1,8 @@
-<soap:Envelope xmlns:soap="http://www.w3.org/2003/05/soap-envelope" xmlns:ns="http://aamva.org/dldv/wsdl/2.1" xmlns:ns1="http://aamva.org/niem/extensions/1.0" xmlns:ns2="http://niem.gov/niem/niem-core/2.0">
+<soap:Envelope
+  xmlns:soap="http://www.w3.org/2003/05/soap-envelope"
+  xmlns:ns="http://aamva.org/dldv/wsdl/2.1"
+  xmlns:aa="http://aamva.org/niem/extensions/1.0"
+  xmlns:nc="http://niem.gov/niem/niem-core/2.0">
```

Comment:
Big fan of the clearer namespaces! Since there's not a character limit, what if we gave them even longer, clearer names?
```suggestion
  xmlns:dldv="http://aamva.org/dldv/wsdl/2.1"
  xmlns:niemext="http://aamva.org/niem/extensions/1.0"
  xmlns:niemcore="http://niem.gov/niem/niem-core/2.0">
```
To extend your table

What we used to call it | What we call it now | New proposal | What it means
-- | -- | -- | --
`ns:` | `ns:` | **`dldv:`** | Elements from DLDV WSDL
`ns2:` | `nc:` | **`niemcore:`** | Elements from NIEM core
`ns1:` | `aa:` | **`niemext:`** | AAMVA-specific extensions to NIEM core

Reply:
Those are better. Actually now that I look at it, they are using `dldv:` in some places in the docs:

![image](https://github.com/18F/identity-idp/assets/364697/b80adad3-652a-42c4-bc9a-0ce0cac46430)

I think I will change `ns` to `dldv` and leave the other to match the docs